android/app/src/sharedTest/java/org/cryptimeleon/incentive/app/data/FakeCryptoRepository.kt

```diff
@@ -52,11 +54,15 @@ class FakeCryptoRepository(
         }
     }
 
-    override suspend fun runCreditEarn(
-        basketId: UUID,
-        promotionParameters: PromotionParameters,
-        basketValue: Int
-    ) {
+    override suspend fun sendTokenUpdatesBatch(basketId: UUID, bulkRequestDto: BulkRequestDto) {
+        TODO("Not yet implemented")
+    }
+
+    override suspend fun retrieveTokenUpdatesResults(basketId: UUID): BulkResponseDto {
+        TODO("Not yet implemented")
+    }
+
+    override suspend fun putToken(promotionParameters: PromotionParameters, token: Token) {
```

Comment:
what exactly does FakeCryptoRepository (not yet)?
These three TODOs are in issues right?